bin/run-in-docker.sh

```diff
@@ -14,8 +14,9 @@ print_help() {
     --help, -h                    Print this help block
 
   Available parameters:
-    S2S_URL         Defaults to 'localhost'
-    STUB_NOTIFY     Defaults to 'true'
+    APPLICATION_INSIGHTS_IKEY     Defaults to 'app-insight-key'
```

Comment:
typo in `IKEY`?

Reply:
that's the primary environment variable name defined in [sdk](https://github.com/Microsoft/ApplicationInsights-Java/blob/v1.0.10/core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java#L62)

Reply:
oh, 'I' for 'instrumentation'.